BuildShop/BuildShopData/Implementations/OrderRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using BuildShopDataAccessLayer.Repositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace BuildShopDataAccessLayer.Implementations
+{
+    public class OrderRepository : IOrderRepository
```

Comment:
it's really strange, that you have base service but you didn't implement base repository